packages/web-components/src/components/ic-toggle-button-group/ic-toggle-button-group.stories.mdx

```diff
@@ -0,0 +1,434 @@
+  component="Toggle Button Group"
+/>
+
+<Description markdown={readme} />
+
+### Select type
+
+<Canvas>
+  <Story name="Select type" parameters={{ loki: { skip: true } }}>
+    {(args) =>
+      html`
+        <span>Single and manual</span>
+        <ic-toggle-button-group select-type="single">
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+        </ic-toggle-button-group>
+        <br />
+        <span>Single and auto</span>
+        <ic-toggle-button-group
+          label="This is a label"
+          name="group1"
+          select-method="auto"
+        >
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+        </ic-toggle-button-group>
+        <br />
+        <span>Multi and manual</span>
+        <ic-toggle-button-group
+          label="This is a label"
+          name="group1"
+          select-type="multi"
+        >
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+        </ic-toggle-button-group>
+      `
+    }
+  </Story>
+</Canvas>
+### Sizes
+
+<Canvas>
+  <Story name="Sizes" parameters={{ loki: { skip: true } }}>
+    {(args) =>
+      html`
+        <span>Default</span>
+        <ic-toggle-button-group label="This is a label" name="group1">
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+          <ic-toggle-button label="Toggle"></ic-toggle-button>
+        </ic-toggle-button-group>
+        <br />
+        <span>Small</span>
+        <ic-toggle-button-group
+          label="This is a label"
+          name="group1"
+          size="small"
+        >
+          <ic-toggle-button label="Toggle" size="small"></ic-toggle-button>
+          <ic-toggle-button label="Toggle" size="small"></ic-toggle-button>
+          <ic-toggle-button label="Toggle" size="small"></ic-toggle-button>
+        </ic-toggle-button-group>
+        <br />
+        <span>Large</span>
+        <ic-toggle-button-group
+          label="This is a label"
+          name="group1"
+          size="large"
+        >
+          <ic-toggle-button label="Toggle" size="large"></ic-toggle-button>
+          <ic-toggle-button label="Toggle" size="large"></ic-toggle-button>
+          <ic-toggle-button label="Toggle" size="large"></ic-toggle-button>
+        </ic-toggle-button-group>
+        <br />
+        <span>Full width</span>
+        <ic-toggle-button-group
+          label="This is a label"
+          name="group1"
+          full-width="true"
+        >
+          <ic-toggle-button label="Toggle" full-width="true"></ic-toggle-button>
```

Comment:
```suggestion
          <ic-toggle-button label="Toggle"></ic-toggle-button>
```
The prop does not need to be repeated in the toggle button as it would inherit it from the toggle button group. Your React stories are fine but it is the web-components stories which have this.